app/src/main/java/org/wikipedia/feed/news/NewsFragment.kt

```diff
@@ -60,6 +60,9 @@ class NewsFragment : Fragment() {
 
         DeviceUtil.updateStatusBarTheme(requireActivity(), binding.toolbar, true)
         binding.appBarLayout.addOnOffsetChangedListener { layout, offset ->
+            if (!isAdded) {
```

Comment:
The crash you linked seems to be specific to the NewsFragment. Is there a reason to add unnecessary checks to the other fragments?

If you compare the way the OffsetChangedListener is used here, versus in ArticleEditDetailsFragment, the real reason for the crash becomes more clear.